gen_templates.py

```diff
@@ -24,7 +24,7 @@
             "difName": "n.DIF"
         },
         {
-            "encodedAppName": "rina.apps.echotime.client-1--",
+            "encodedAppName": "traffic.generator.server-1--",
             "difName": "n.DIF"
```

Comment:
The thing is that only "server" applications are needed in the da.map (so echotime clients can be removed). In fact, I would just have the echo time server and the tgen server there.. Updating the PR